controller/eks-cluster-config-handler.go

```diff
@@ -1151,6 +1238,13 @@ func (h *Handler) importCluster(config *eksv1.EKSClusterConfig, eksService *eks.
 		}
 	}
 
+	launchTemplatesOutput, err := ec2Service.DescribeLaunchTemplates(&ec2.DescribeLaunchTemplatesInput{
+		LaunchTemplateNames: []*string{aws.String(fmt.Sprintf(launchTemplateNameFormat, config.Spec.DisplayName))},
```

Comment:
could we actually somehow tag/label the nodegroups so we know Rancher owns them instead of checking the name? or do both? tagging the resource could also be used for the user or aws admin.

Reply:
This is looking for the launch template with the prescribed name, not the nodegroup.

We do also tag the launch template when it is created. The problem in this case is: if a launch template with this name already exists, then Rancher won't be able to create one. Therefore, Rancher takes over ownership of the launch template so it can create new versions of it.

Reply:
Sounds good.